packages/react/src/components/Pagination/Pagination.tsx

```diff
@@ -184,7 +130,8 @@ Pagination.propTypes = {
   onLastPageClick: PropTypes.func,
   // @ts-expect-error
   tooltipPlacement: PropTypes.string,
-  className: PropTypes.string
+  className: PropTypes.string,
+  thin: PropTypes.bool
```

Comment:
Not part of this PR, but a linting issue was raised with `thin` not being defined on prop types